src/chapters/enums/enums-in-models.rst

```diff
@@ -14,3 +14,241 @@ The final code from this video is in the `enums branch <https://github.com/Launc
 
 Enum Types in Models - Text
 ---------------------------
+
+Create an Enum Class
+^^^^^^^^^^^^^^^^^^^^
+
+In your ``models`` package within ``coding-events``, create a new class called ``EventType``. 
+Before you finish entering the name of your file, select the ``Enum`` option from the list of 
+types.
+
+.. figure:: figures/enumTypeSelection.png
+   :scale: 50%
+   :alt: Enum type selection.
+
+Enum type selection.
+
+Because enum values are constants, we use :ref:`naming-conventions` and write them in all caps.
+Each value is demarcated with a comma and the list is completed with a semicolon.
+
+``EventType``:
+
+.. sourcecode:: java
+   :lineno-start: 6
+
+   public enum EventType {
+
+      CONFERENCE,
+      MEETUP,
+      WORKSHOP,
+      SOCIAL;
+
+   }
+
+Add an Enum Field
+^^^^^^^^^^^^^^^^^
+
+Enums can store additional information affiliated with each value in properties. Enum properties 
```

Comment:
This paragraph conflates fields (class member variables) with properties (fields made available via accessor methods). Either term works, given the correct context and agreement across the paragraph.